mgmt/rest/common/common.go

```diff
@@ -0,0 +1,39 @@
+package common
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"runtime"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/intelsdi-x/snap/control"
+)
+
+func WriteFile(filename string, b []byte) (string, error) {
+	// Create temporary directory
+	dir, err := ioutil.TempDir(control.GetDefaultConfig().TempDirPath, "snap-plugin-")
```

Comment:
The trailing isn't required. Each time a plugin loads, a tmp file is being generated with a random number. snap-plugin- would be the prefix with which the file is generated. It might as well just be snap-plugin.